packages/genesys-spark-components/src/components/stable/gux-listbox/options/gux-option/gux-option.tsx

```diff
@@ -71,13 +73,22 @@ export class GuxOption {
     return this.selected ? 'true' : 'false';
   }
 
+  private hasDisabledParent(): boolean {
+    const parentListbox = getClosestElement(
+      'gux-listbox',
+      this.root
+    ) as HTMLGuxListboxElement;
+
+    return parentListbox?.disabled || false;
+  }
```

Comment:
Could this be  `return parentListbox?.disabled` as the `disabled` prop would be initially `false` anyways I think.

Reply:
Fixed

Reply:
Fixed